Is this a bug in the workflow, or am I supposed to do something? I do have "allow edits by maintainers" checked.
```
Run git config user.name "dseomn" &&
[cmdline-no-secrets 10adce5] Convert Markdown to HTML
 1 file changed, 1 insertion(+), 1 deletion(-)
remote: Permission to dseomn/gitgitgadget.github.io.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/dseomn/gitgitgadget.github.io/': The requested URL returned error: 403
Error: Process completed with exit code 128.
```

--- a/.github/workflows/pr.yml
+++ b/.github/workflows/pr.yml
@@ -6,6 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       contents: write
+      issues: write
     steps:
       - uses: actions/checkout@v4
         with:
@@ -19,7 +20,7 @@ jobs:
           git add -A \*.html &&
           git diff-index --cached --exit-code HEAD -- ||
           echo "need-to-commit=yes" >>$GITHUB_OUTPUT
-      - name: commit & push
+      - name: commit
         if: steps.check.outputs.need-to-commit == 'yes'
         run: |
           git config user.name "${{github.actor}}" &&
@@ -28,5 +29,21 @@ jobs:
           git update-index --refresh &&
           git diff-files --exit-code &&
           git diff-index --cached --exit-code HEAD -- &&
-          git push
-
+          git bundle create git.bundle ${{ github.event.pull_request.head.sha }}..${{ github.event.pull_request.head.ref }}
+      - name: commit
+        if: steps.check.outputs.need-to-commit == 'yes' && github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name
+        run: git push
+      - name: publish bundle
+        id: bundle
+        if: steps.check.outputs.need-to-commit == 'yes' && github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name
+        uses: actions/upload-artifact@v4
+        with:
+          name: bundle
+          path: git.bundle
+      - name: instructions how to fetch bundle
+        if: steps.check.outputs.need-to-commit == 'yes' && github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name
+        run: |
+          body='The HTML pages are out of date. Please download the `bundle` artifact from ${{ steps.bundle.outputs.artifact-url }}, extract the `git.bundle` file from it, then run `git pull /path/to/git.bundle ${{ github.event.pull_request.head.ref }}` and then push to the PR branch'
+          echo "::error::$body" >&2
+          echo "$body" >>$GITHUB_STEP_SUMMARY
+          exit 1

--- a/reply-to-this.html
+++ b/reply-to-this.html
@@ -33,7 +33,7 @@
 <p>Simply copy the file to the <code>new</code> subfolder in your mailer&#39;s maildir folder.</p>
 <h3 id="if-you-use-webmail-gmail-and-friends">If you use webmail (GMail and friends)</h3>
 <p>You can use the command-line tool <code>curl</code> (provided that your version has IMAP support):</p>
-<pre><code class="language-sh">curl -g --user &quot;&lt;email&gt;:&lt;password&gt;&quot; --url &quot;imaps://imap.gmail.com/INBOX&quot; -T /path/to/raw.txt
+<pre><code class="language-sh">curl -g --user &quot;&lt;email&gt;&quot; --url &quot;imaps://imap.gmail.com/INBOX&quot; -T /path/to/raw.txt
 </code></pre>
 
     </div>

